pkgs/by-name/yf/yffi/package.nix

```
@@ -0,0 +1,55 @@
+{
+  fetchFromGitHub,
+  lib,
+  rust-cbindgen,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "yffi";
+  version = "0.24.0";
+
+  src = fetchFromGitHub {
+    owner = "y-crdt";
+    repo = "y-crdt";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-RFdLEsKQxt8BGqbf5FFM7mZD64glW7bTKRJaAQOe+vo=";
+  };
+
+  cargoHash = "sha256-RL9lBc7lSmc6jOavE5lZupmaNGZgQhJBedNhjfHVajg=";
+
+  buildAndTestSubdir = "yffi";
+
+  nativeBuildInputs = [
+    rust-cbindgen
+  ];
+
+  preBuild = ''
+    cbindgen --config ./yffi/cbindgen.toml --crate yffi --output ./tests-ffi/include/libyrs.h --lang C
```

Can't we just output `libyrs.h` in `$out/include/`?

```suggestion
    cbindgen --config ./yffi/cbindgen.toml --crate yffi --output $out/include/libyrs.h --lang C
```

The tests should pick it up with `-I $out/include`.